src/Template/Sadrs/xml/e2b.ctp

```diff
@@ -69,8 +71,8 @@
         ?></medicallyconfirm>
         <?php $arr = preg_split("/[\s]+/", $sadr['reporter_name']); ?>
         <primarysource>
-            <reportergivename><?php if (isset($arr[0])) echo $arr[0]; ?></reportergivename>
-            <reporterfamilyname><?php if (isset($arr[1])) echo $arr[1].' '; if (isset($arr[2])) echo $arr[2];  ?></reporterfamilyname>
+            <reportergivename><?php if (isset($arr[0])) echo preg_replace('/[^A-Za-z0-9\s+]/', '',$arr[0]); ?></reportergivename>
+            <reporterfamilyname><?php if (isset($arr[1])) echo preg_replace('/[^A-Za-z0-9\s+]/', '',$arr[1]).' '; if (isset($arr[2])) echo preg_replace('/[^A-Za-z0-9\s+]/', '',$arr[2]);  ?></reporterfamilyname>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate